src/functionalTest/java/submitBulk/ClaimStatusUpdate.java

```diff
@@ -0,0 +1,65 @@
+import net.serenitybdd.junit.runners.SerenityRunner;
+import net.thucydides.core.annotations.Steps;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import io.restassured.path.xml.XmlPath;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.matchesRegex;
+import static org.hamcrest.CoreMatchers.containsString;
+import org.junit.jupiter.api.Assertions;
+import steps.ClaimStatusSteps;
+
+
+@RunWith(SerenityRunner.class)
```

Comment:
This should be replaced with the JUnit5 equivalent which is `@ExtendWith(SerenityJUnit5Extension.class)`